components/src/hardware-sim/Labware/labwareInternals/Tips/UsedTip.tsx

```diff
@@ -0,0 +1,24 @@
+export function UsedTip(props: { size?: string | number }): JSX.Element {
+  const { size } = props
+  const width = size ?? '20'
+  const height = size ?? '20'
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 20 20"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle cx="10" cy="10" r="10" fill="#BFDCFD" />
+      <circle
+        cx="10"
+        cy="10"
+        r="4.5"
+        fill="#737578"
+        stroke="#737578"
```

Comment:
```suggestion
        fill={COLORS.grey50}
        stroke={COLORS.grey50}
```

maybe just import `grey50`